engine/src/main/java/org/terasology/input/InputSystem.java

```diff
@@ -284,7 +284,7 @@ public void update(float delta) {
     }
 
     public boolean isCapturingMouse() {
-        return capturingMouse && display.hasFocus();
+        return true;
```

Comment:
Although maybe I could also point this out. It appears to have the side effect that if you de-focus the window you can move the mouse cursor over it (still without giving the window focus) and the game will react. Previously the game entirely ignored input while not having focus. Dunno if we'd want that change?